math/src/main/java/com/powsybl/math/matrix/DenseMatrix.java

```diff
@@ -224,6 +212,29 @@ public void print(PrintStream out, List<String> rowNames, List<String> columnNam
         }
     }
 
+    private int getMaxWidthAmongRowNames(List<String> rowNames) {
```

Comment:
getMaxRowNamesLength?
I'm not sure width is the good word here. If you agree, also change the names of the variables

Reply:
Here we really try to find the max width of first column so I think width is OK.